classroom/snippets/src/main/java/ModifyAttachmentsStudentSubmission.java

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// [START classroom_modify_attachments_student_submissions_class]
+
+import com.google.api.client.googleapis.json.GoogleJsonError;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.classroom.Classroom;
+import com.google.api.services.classroom.ClassroomScopes;
+import com.google.api.services.classroom.model.Attachment;
+import com.google.api.services.classroom.model.Link;
+import com.google.api.services.classroom.model.ModifyAttachmentsRequest;
+import com.google.api.services.classroom.model.StudentSubmission;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
+/* Class to demonstrate the use of Classroom ModifyAttachments StudentSubmissions API. */
+public class ModifyAttachmentsStudentSubmission {
+  /**
+   * Lists student submissions based on courseId, courseWorkId, and userId.
+   *
+   * @param courseId - identifier of the course.
+   * @param courseWorkId - identifier of the course work.
+   * @param id - identifier of the student submission.
+   * @return - the modified student submissions.
```

Comment:
should this just be "submission" singular?

Reply:
also consider making it consistent between examples - this is a "modified student submission" but the patch example returns an "updated StudentSubmission instance".